client/gutenberg/extensions/simple-payments/editor.js

```diff
@@ -0,0 +1,39 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { registerBlockType } from '@wordpress/blocks';
+import GridiconMoney from 'gridicons/dist/money';
+
+registerBlockType( 'a8c/simple-payments', {
```

Comment:
I think we should prefix all the blocks with jetpack instead of a8c. cc: @MichaelArestad

Reply:
I agree with you. Let's make sure they're all Jetpack.

Reply:
https://github.com/Automattic/wp-calypso/pull/28043